### PR TITLE
aruha-355: changed api according to guidlines and task validation

### PR DESCRIPTION
**Pull request by adyach**
I would like to send this api to guild but first it will be nice to have an agreement within the team. So you are welcome to comment.

### CODE REVIEW

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -1054,7 +1082,7 @@ paths:
                   $ref: '#/definitions/Feature'
             required:
               - items
-    post:
+    put:
```
Comment (antban) [inline api/nakadi-event-bus-api.yaml]: I don't think that it's a good idea

**Comment by rcillo** [comment]
👍 looks great.

**Comment by v-stepanov** [comment]
I actually thought that the /settings is our internal API and it should not be in swagger file
If we want to expose that then I'm strongly against word "flooder" it's misleading.

**Comment by v-stepanov** [comment]
Looks good for me now

**Comment by v-stepanov** [comment]
👍

**Comment by v-stepanov** [comment]
👍

**Comment by rcillo** [comment]
👍

**Pull request merged**